### PR TITLE
feat(video): video details page

### PR DESCRIPTION
**Pull request by natali-abyss**

### CODE REVIEW

**Comment by rychkog** [comment]
LGTM

File: app/sections/videos/videos-list.component.styl [hunk 1]
```diff
@@ -27,3 +30,6 @@
       left 17px
       position absolute
 
+.post-title
+  margin 5px 0
+  display block
```
Comment (dydek) [inline app/sections/videos/videos-list.component.styl]: new line

File: app/shared/components/entries-view/entries-html.styl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import "../../../variables.styl"
+
+.block-entry
+  h2
+    font-size 28px
+  a
+    color color-yellow
+    text-decoration underline
```
Comment (dydek) [inline app/shared/components/entries-view/entries-html.styl]: new line

File: app/sections/videos/video-details.component.styl [hunk 3]
```diff
@@ -0,0 +1,3 @@
+h1
+  font-size 38px
+  margin-bottom 0
```
Comment (dydek) [inline app/sections/videos/video-details.component.styl]: new line

**Pull request merged**